app/databaseAccessors/articleAccessor.js

```diff
@@ -307,4 +307,27 @@ export default class ArticlesAccessor {
       ErrorInternalAPIModelFieldValidation(e);
     }
   }
+
+    /**
+   * updateArticle method
+   *
+   * updates the article with the parameter provided
+   *
+   * @param {string} slug
+   * @param {JSON} update
+   * @returns updated article
+   */
+    static async updateArticle(slug, update) {
+      try {
+        await Connection.open();
+        const article = await Article.findOneAndUpdate({ slug }, update, { new: true });
```

Comment:
This will likely be a problem when you validate the outgoing response as an ArticleResponse, but you will need to populate this query so that we aren't just returning ObjectIds with no other data as subfields. 

If I remember correctly, it will be as follows:
```suggestion
        const article = await Article.findOneAndUpdate({ slug }, update, { new: true }).populate("users").exec();
```